src/blockdata/script.rs

```diff
@@ -121,9 +121,16 @@ impl core::str::FromStr for Script {
 }
 
 /// An object which can be used to construct a script piece by piece.
-#[derive(PartialEq, Eq, Debug, Clone)]
+#[derive(PartialEq, Eq, Clone)]
 pub struct Builder(Vec<u8>, Option<opcodes::All>);
-display_from_debug!(Builder);
+
+impl fmt::Display for Builder {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        let stack_content = std::str::from_utf8(&self.0).unwrap();
+        writeln!(f, "stack : {}, opCode: {}",stack_content, self.1.as_ref().unwrap())
```

Comment:
Firstly, I think this should be displayed the same as `Script`.

Secondly, this shows why this struct should've had named fields, since you got confused about what they mean. The `0` field is just script, `1` is last opcode or `None` if last thing was a push. This is also why it's wrong to `unwrap` - it'd just panic in most cases!

Reply:
Thank you for clarifying the confusion, i updated to implementation to be identical to Script.